lib/rules/__tests__/index.test.mjs

```diff
@@ -74,36 +74,39 @@ describe('custom message option', () => {
 });
 
 describe('standard config', () => {
-	const tmpDir = path.join(__dirname, 'tmp');
-
-	// NOTE: The use of Promised-based APIs may cause flaky test on CI.
-	rmSync(tmpDir, { recursive: true, force: true });
-	mkdirSync(tmpDir, { recursive: true });
-	writeFileSync(path.join(tmpDir, 'package.json'), '{}');
-	execSync(
-		'npm install --silent --no-package-lock --no-audit --omit=peer stylelint-config-standard',
-		{ cwd: tmpDir },
-	);
+	// eslint-disable-next-line jest/no-disabled-tests
```

Comment:
[suggest] How about leaving a reason to skip it?

For example:

```suggestion
	// eslint-disable-next-line jest/no-disabled-tests -- To prevent a failure when adding a new rule to the sharable config. See #7045.
```